tests/system/fim/roles/aws/tasks/generate_environment_template.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
This file is not required anymore, please delete it.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/482/commits/a42a0040017e5cc3755ad594fa757f0e55198723 .